.editorconfig

```diff
@@ -0,0 +1,16 @@
+# editorconfig.org
```

Comment:
Please change the commit messsage to lower case.

At the moment, it isn't being validated correctly because of a bug in commitlint.